kalite/__init__.py

```diff
@@ -7,4 +7,11 @@
 __version__ = VERSION
 
 # ROOT_DATA_PATH is *not* where the source files live. It's a place where non-user-writable data files may be written.
-ROOT_DATA_PATH = os.path.join(sys.prefix, 'share', 'kalite')
+ROOT_DATA_PATH = None
```

Comment:
Can you remove this initialization of `ROOT_DATA_PATH`, and move the comment closer to the `if` statement? That makes it easier to find where `ROOT_DATA_PATH` is meaningfully set.